katello/pulp-katello/pulp-katello.spec

```diff
@@ -0,0 +1,120 @@
+# Copyright (c) 2013 Red Hat, Inc.
+#
+# This software is licensed to you under the GNU General Public
+# License as published by the Free Software Foundation; either version
+# 2 of the License (GPLv2) or (at your option) any later version.
+# There is NO WARRANTY for this software, express or implied,
+# including the implied warranties of MERCHANTABILITY,
+# NON-INFRINGEMENT, or FITNESS FOR A PARTICULAR PURPOSE. You should
+# have received a copy of GPLv2 along with this software; if not, see
+# http://www.gnu.org/licenses/old-licenses/gpl-2.0
+
+
+%{!?python_sitelib: %global python_sitelib %(%{__python} -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())")}
+%{!?python_sitearch: %global python_sitearch %(%{__python} -c "from distutils.sysconfig import get_python_lib; print(get_python_lib(1))")}
+
+
+Name: pulp-katello
+Version: 1.0.2
+Release: 1%{?dist}
+Summary: Plugins useful for katello interactions with pulp
+Group: Development/Languages
+License: GPLv2
+URL: https://fedorahosted.org/katello/
```

Comment:
Should this be katello.org?

Reply:
Updated to just point at github repository.